account/identity.go

```diff
@@ -58,11 +58,17 @@ func (u NullUUID) Value() (driver.Value, error) {
 // One User account can have many Identities
 type Identity struct {
 	gormsupport.Lifecycle
-	ID       uuid.UUID `sql:"type:uuid default uuid_generate_v4()" gorm:"primary_key"` // This is the ID PK field. For identities provided by Keyclaok this ID equals to the Keycloak. For other types of IDP (github, oso, etc) this ID is generated automaticaly
-	Username string    // The username of the Identity
-	Provider string    // The identity provider ID, such as "keycloak", "github", "oso", etc
-	UserID   NullUUID  `sql:"type:uuid"` // Belongs to User
-	User     User
+	// This is the ID PK field. For identities provided by Keyclaok this ID equals to the Keycloak. For other types of IDP (github, oso, etc) this ID is generated automaticaly
+	ID uuid.UUID `sql:"type:uuid default uuid_generate_v4()" gorm:"primary_key"`
+	// The username of the Identity
+	Username string
+	// ProviderType The type of provider, such as "keycloak", "github", "oso", etc
+	ProviderType string `gorm:"column:provider_type"`
```

Comment:
Yep! I understood why you named `providerType` but I mentioned that there is assignation to provider == providerType for some objects like my comment in here: https://github.com/almighty/almighty-core/pull/777#discussion_r102270979

I am fine with both as long as they are consistent everywhere :).